SmRtAPI/SmRtAPI/V2/MessageReader.cs

```diff
@@ -70,53 +70,54 @@ private bool ProcessMessage(WebSocketReceiveResult result, ArraySegment<byte> me
             switch (jsonObject.Value<string>("message"))
             {
                 case "RecognitionStarted":
```

Comment:
Can you revert these formatting changes?

Reply:
will do.